bin/parse_go_sum

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env ruby
+
+require 'json'
+
+parsed_file = File.read(ARGV[0])
+
+dependencies = []
+
+parsed_file.each_line do |line|
+  # Split on space
+  # Example - github.com/google/go-cmp v0.5.3/go.mod h1:v8dTdLbMG2kIc/vJvl+f65V22dbkXbowE6jgT/gNBxE=
+  # Validate 3 parts - name, version and checksum
+  parts = line.split
+  if parts.length == 3
+    names = parts[0].split("/")
+    dependencies.append(
+      {
+        "namespace" => names[0..-2].join("/").to_s,
+        "name" => names[-1].to_s,
+        "version" => parts[1].to_s.gsub('/go.mod', ''),
+        "checksum" => parts[2].to_s.gsub('h1:', '')
```

Comment:
what is `h1`? Can it ever have non-h1 values?

Reply:
the checksum line will have issues if the checksum itself contains `h1`.

Reply:
Not that I have seen, h1: is always there - 
Example - `cloud.google.com/go v0.22.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=`

Reply:
Also, won't the gsub just match `"h1:"`, so `h1` in the checksum shouldn't be affected